content/articles/compiled-redesign.md

```diff
@@ -0,0 +1,141 @@
+years later with more experience and grounded understanding, I redesigned
+_CompilED_ as an exercise to develop a website that conforms very well to the
+WCAG. This experience led me to best practices and a workable flow that I can
+share with my team for other development processes.
+
+## Steps on improving CompilED’s accessibility
+
+### User experience design
+
+I thought carefully about _CompilED_ content delivery and comparable user
+experience on different medium, from visual to auditory experience. Much of the
+UX design centered around the integrity of the visual UI and the browser’s
+rendering of the accessibility tree for screen readers. The elements and
+content on the pages were regrouped according to function, reordered according
+to priority, and restacked for responsive design. This regrouping and
+reordering was done so that the structure on each page make sense to all users
+regardless of their choice of devices—desktop, mobile, or assistive
+technologies.
+
+I’ve written on similar design process for the
+[redesign of PHTC image map](/articles/a11y-rwd-imagemap/) and
+[two Dental School projects](/articles/deconstructing-accessibility/).
+
+### Semantic HTML and ARIA
+
+In her demonstration on 
+[how a screen reader user accesses the web](https://www.smashingmagazine.com/2019/02/accessibility-webinar/),
+accessibility engineer [Léonie Watson](https://tink.uk/) recommended using
+semantic HTML and ARIA attributes to improve content accessibility and clarify
+meaning for content groups. They serve as sign posts to help users to chart a
+mental model of a page. I was already using HTML5 landmark regions (`header`,
+`nav`, `main`, `section`, `article`, `footer`) in _CompilED_. This time, I paid
+more attention to the ARIA labels to identify components on the pages so that
+they can be read by screen readers. 
+
+### Keyboard accessibility
+
+Keyboard navigation is markedly improved for _CompilED_ pages—all functionality
+and interactive elements are available from a keyboard. I also redesigned the
+visual focus styling (`hover`, `focus`, `active`), thanks to the
+[helpful tips from Eric Bailey.](https://www.youtube.com/watch?v=9YazmVNwtHI)
+
+### Alt text for images and social media
+
+While it is common knowledge that alternative text for images is necessary,
+what is less know is what sort of description should be used for the `alt`
+text. To put simply, image description depends on context.
+
+All images on _CompilED_ articles how have `alt` attribute built in the
+template. It is up to the authors to decide how the images are used to provide
+context in the `alt` text. W3C provides a good
+[tutorial on image concepts](https://www.w3.org/WAI/tutorials/images/).
+WebAIM also has a
+[thorough explanation on alternative text](https://webaim.org/techniques/alttext/)
+and images.
+
+_CompilED_ also uses `og:image:alt` and `twitter:image:alt` structured
+properties to provide descriptions images used for social media sharing.
```

Comment:
maybe '...descriptions for images shared via social media.'

Reply:
I rewrote the sentence.